be/src/runtime/mem_tracker.cpp

```diff
@@ -184,6 +184,14 @@ MemTracker* MemTracker::CreateQueryMemTracker(const TUniqueId& id,
 }
 
 MemTracker::~MemTracker() {
+    int64_t remaining_bytes = consumption();
+    // work around some scenario where consume() is not paired with release()
+    // e.g., in the initialization of hll and bitmap aggregator (see aggregate_func.h)
+    if (remaining_bytes > 0) {
+        for (auto tracker : _all_trackers) {
+            tracker->_consumption->add(-remaining_bytes);
+        }
+    }
```

Comment:
Actully, in normal case, when a MemTracker destructs, the consumption should be 0. The memory user may release the memory usage to MemTracker later, which will cause a process crash. So better to comment the case that aquire and release is not in pair as TODO, it should be fixed in the future.

Reply:
Agreed. The invalid use of MemTracker::consume in aggregate_func.h should be fixed and we should replace the code here with `DCHECK_EQ(consumption(), 0)`